src/pages/lifetime/celebration/singlesday/[filename].tsx

```diff
@@ -0,0 +1,182 @@
+import PlatformSection from '@/components/black-friday/PlatformSection';
+import { TextAndCardsGroupColumnSection } from '@/components/shared/components/TextAndCardsGroupColumnSection';
+import Image from 'next/image';
+import { Eye, Fingerprint, LockKey, ShieldCheck } from '@phosphor-icons/react';
+import { FooterText, MetatagsDescription, NavigationBarText } from '@/assets/types/layout/types';
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+
+interface SinglesDayCelebrationTemplateProps {
+  lang: string;
+  langJson: SinglesDay;
+  navbarLang: NavigationBarText;
+  footerLang: FooterText;
+  metatagsDescriptions: MetatagsDescription[];
+  pathname:string
+}
+const ALLOWED_PATHS = [
+  'en',
+  'tw',
+  'ch',
+];
+
+const SinglesdayCelebrationTemplate = ({
+  lang,
+  langJson,
+  navbarLang,
+  footerLang,
+  metatagsDescriptions,
+  pathname
+ 
+}: SinglesDayCelebrationTemplateProps): JSX.Element => {
+const metatags = metatagsDescriptions.filter((desc) => desc.id === 'singles-day');
+const discount=0.2;
+const percent='80%'
+const currencySpecified='US';
+const locale = lang as string;
+const couponCode=PromoCodeName.SinglesDay;
+const router = useRouter();
+const filename = pathname.split('/').pop();
+console.log('SinglesDay Template - Filename:', pathname);
+const selectedPathName = ALLOWED_PATHS.find((allowedPathname) => allowedPathname === `${filename}`);
+
+useEffect(() => {
+  console.log('FILENAME: ', filename);
+  if (!selectedPathName) {
+    router.push('/lifetime');
+  }
+}, [selectedPathName, router]);
+
+
+const Cards = [
+    {
+      icon: ShieldCheck,
+      title: langJson.FeatureSection.cards[0].title,
+      description: langJson.FeatureSection.cards[0].description,
+    },
+    {
+      icon: LockKey,
+      title: langJson.FeatureSection.cards[1].title,
+      description: langJson.FeatureSection.cards[1].description,
+    },
+    {
+      icon: Eye,
+      title: langJson.FeatureSection.cards[2].title,
+      description: langJson.FeatureSection.cards[2].description,
+    },
+    {
+      icon: Fingerprint,
+      title: langJson.FeatureSection.cards[3].title,
+      description: langJson.FeatureSection.cards[3].description,
+    },
+  ];
+  return (
+    <Layout
+      title={metatags[0].title}
+      description={metatags[0].description}
+      segmentName="SinglesDay"
+      lang={lang}
+      specialOffer={`https://internxt.com/images/previewLink/LifetimePreviewLink.png`}
+    >
+    <Navbar textContent={navbarLang} lang={locale}  cta={['default']} fixed mode="payment" isLinksHidden hideNavbar />
+     <div className="bg-gray-1">
+     
+   <HeroSection 
+        TextComponent={
+        <div className="flex flex-col items-center justify-center text-center md:text-left md:items-start md:justify-start">
+          <p className='inline-block text-xl font-medium text-gray-80 bg-gray-10 p-3 mb-8 rounded-md text-center'>{langJson.HeroSection.offer}</p>
+          <div className='mb-8'>
```

Comment:
Use `space-y-8/space-x-8/gap-8` in the parent instead. It is better to work with general spacing for all divs.